src/NServiceBus.RabbitMQ/Connection/ConnectionFactory.cs

```diff
@@ -30,7 +30,8 @@ public ConnectionFactory(ConnectionConfiguration connectionConfiguration)
                 Password = connectionConfiguration.Password,
                 RequestedHeartbeat = connectionConfiguration.RequestedHeartbeat,
                 AutomaticRecoveryEnabled = true,
-                NetworkRecoveryInterval = connectionConfiguration.RetryDelay
+                NetworkRecoveryInterval = connectionConfiguration.RetryDelay,
+                UseBackgroundThreadsForIO = true,
```

Comment:
I did end up making that change in relation to working around the problem with the V5 acceptance test framework, but I think it's an OK change to make regardless. It's actually rather odd that the connection threads aren't background threads by default anyway.